index.html

```diff
@@ -1 +1,55 @@
-Chris McCourt
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Chris's webpage </title>
+    <link href="css/style.css" rel="stylesheet"> 
+  </head>
+  <body>
+
+    <h1>Welcome to my page!</h1>
+
+    <h2>Chris McCourt</h2>
+
+    <h3>About</h3>
+
+      <p>My name is Chris. 
+       The best time of my life has been right now. Being out of work and taking care of my two kids has only furthered my desire to become a better more experienced human being.
+       My interest in coding really took off when I created a family vacation app using Appsheets in 2018. 
+       It used API's to grab local weather information and it used google maps data to do in app searches for things like local breakfast spots and cafes etc. 
+       The low code experience was great, but I really wanted to challenge myself to build something from ground up. So, I started looking into coding and got hooked.
+       
+      <p> Now, I am a current student of Code The Dream and on my way to becoming a junior software developer. 
+       I am excited to use these new skills in a way that will inspire others around me. 
+       My life motto is : “If  I can learn to do it, then so can you”. </p>
+
+    <h3>Experience</h3>
+
+    <h4>Work History </h4>
+        <p>File Clerk @ Cardiologist Office</p>
```

Comment:
Chris, I will suggest using unordered or ordered lists (up to your preference) here instead of repeating <p> elements.